src/App.tsx

```diff
@@ -8,27 +12,105 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { Todo } from './types/Todo';
+
+import { normalize } from './helpers/normalize';
+
+import { getTodos } from './api';
+
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [query, setQuery] = useState('');
+  const [status, setStatus] = useState('all');
```

Comment:
let's use `TodoStatus` enum 


```suggestion
  const [status, setStatus] = useState(TodoStatus.all);
```